sovrin_common/types.py

```diff
@@ -167,8 +169,9 @@ class ClientPoolUpgradeOperation(MessageValidator):
         (ACTION, NonEmptyStringField()),  # TODO check actual value set
         (VERSION, NonEmptyStringField()),
         # TODO replace actual checks (idr, datetime)
-        (SCHEDULE, MapField(NonEmptyStringField(), NonEmptyStringField(), optional=True)),
-        (SHA256, NonEmptyStringField()),
+        (SCHEDULE, MapField(NonEmptyStringField(),
+                            NonEmptyStringField(), optional=True)),
```

Comment:
I guess the schedule is **\<identifier\>: \<iso time\>**. It makes sense to check more precisely the fields here if it is in the scope of your task.